src/com/pentalog/pentastagiu/homework2/bankaccount/impl/BankAccountImpl.java

```diff
@@ -0,0 +1,36 @@
+package com.pentalog.pentastagiu.homework2.bankaccount.impl;
+
+import com.pentalog.pentastagiu.homework2.bankaccount.BankAccount;
+import com.pentalog.pentastagiu.homework2.exceptions.InsufficientMoneyException;
+import com.pentalog.pentastagiu.homework2.exceptions.NegativeDepositAmountException;
+
+public class BankAccountImpl implements BankAccount {
```

Comment:
desi se intampla si pe proiecte din productie si e o discutie continua legata de asta, nu e recomandat sa foloseti impl in numele pachetului sau al clasei: as sugera mai degraba denumirea Account pt interfata si DepositAccount pt implementare, putand la un moment dat sa extinzi si sa adaugi si un CreditAccount cu o implementare diferita.